File: Meets/Domain/MeetingStatus.cs
namespace Meets.Domain
{
    public enum MeetingStatus
    {
        /// <summary>
        /// Приглашение
        /// </summary>
        Invite,

        /// <summary>
        /// Обсуждение
        /// </summary>
        Discussion,

        /// <summary>
        /// Подтверждено
        /// </summary>
        Confirmed,

        /// <summary>
        /// Завершено
        /// </summary>
        Canceled
    }
}

File: Meets/ClientApp/src/common/MeetingStatus.ts
interface IMeetingStatusItem {
    Title: string
    Code: string
}

export type MeetingStatusItems = 'Invite' | 'Discussion' | 'Confirmed' | 'Canceled';

export const MeetingStatus: Record<MeetingStatusItems, IMeetingStatusItem> = {
    Invite: {
        Title: 'Invite',
        Code: 'Invite'
    },
    Discussion: {
        Title: 'Discussion',
        Code: 'Discussion'
    },
    Confirmed: {
        Title: 'Confirmed',
        Code: 'Confirmed'
    },
    Canceled: {
        Title: 'Canceled',
        Code: 'Canceled'
    }
}